farmdata2/farmdata2_modules/resources/BannerMessageConstants.js

```diff
@@ -0,0 +1,35 @@
+// Declaring regular expression variables so that is
+// defined in all the pages served from out modules.
+
+
+/**
+ * Map of common banner messages used in the BannerComponent for displaying messages
+ * 
+ * <p><b>Banner Message Variables</b></p>
+ * <table>

```

Comment:
I think we can use in-code JSDOC comments to document these values instead of building a separate table.  See some of the code in the `FarmOSAPI.js` file and see if we can place comments with each message so that we don't  have to duplicate all of the content in the code and in the table in the comments.

Reply:
Sounds good, I'll look into it.